habitat/plan.sh

```diff
@@ -32,6 +29,9 @@ pkg_deps=(
   core/libarchive
   core/coreutils
   core/git
+  ngupta26/cookstyle
+  ngupta26/berkshelf
+  ngupta26/chef-cli
```

Comment:
similarly for the origins of these packages?

Reply:
Yeah these too we will be pulling from chef/ origin like chef/cookstyle, chef/chef-cli etc